src/main/java/org/jenkinsci/test/acceptance/plugins/matrix_auth/MatrixAuthorizationStrategy.java

```diff
@@ -20,7 +20,7 @@ public MatrixAuthorizationStrategy(GlobalSecurityConfig context, String path) {
      * Adds a new user to this matrix.
      */
     public MatrixRow addUser(String name) {
-        runThenHandleAlert(() -> this.name.resolve().findElement(by.xpath("../div/span/span/button | ../button")).click(),
+        runThenHandleAlert(() -> this.table.resolve().findElement(by.xpath("../div/span/span/button[text()='Add user\u2026'] | ../div/button[text()='Add user\u2026']")).click(),
```

Comment:
`../button` was from some previous version no longer required so removed.

Added the text as there is also an "add group" button (not that we use it currently but seems like we should be a bit more precice about what we are attempting to obtain.

alas we can not use a `Control` here as the element is not a form element.

Reply:
> some previous version no longer required

Probably before 3.0, which split one "Add user or group" button into two.